src/main/java/com/cloudbees/maven/license/CompleterDelegate.java

```diff
@@ -104,7 +104,6 @@ public void accept(String name) {
                 return;
             }
         }
-        IllegalStateException error = new IllegalStateException("Expecting " + name + " but found " + toString(licenses) + " for dependency " + toString(dependency));
```

Comment:
Just cleanup?

Reply:
it didn't do anything, spotbugs marked this as a 'high' bug